test/withRTLExtension_test.js

```diff
@@ -0,0 +1,151 @@
+/* eslint-disable react/jsx-filename-extension */
+import React from 'react';
+import ReactDOMServer from 'react-dom/server';
+import PropTypes from 'prop-types';
+import { expect } from 'chai';
+import * as aphrodite from 'aphrodite';
+import withRTLExtension from '../src/withRTLExtension';
+
+const { StyleSheet, StyleSheetServer, css } = withRTLExtension(aphrodite);
+
+function MyComponent({ styles }) {
+  return (
+    <div className={css(styles.container)}>Hello World</div>
+  );
+}
+
+MyComponent.propTypes = {
+  styles: PropTypes.object.isRequired, // eslint-disable-line react/forbid-prop-types
```

Comment:
why not import `...withStylesPropTypes` from react-with-styles here?

Reply:
I was following the pattern here - https://github.com/airbnb/react-with-styles/blob/master/test/withStyles_test.jsx#L159

Reply:
ah. both places should probably use withStylesPropTypes then :-)